python/ray/autoscaler/gcp/tpu.yaml

```diff
@@ -51,15 +51,18 @@ head_node_type: ray_head_default
 # Compute instances have python 3.7, but TPUs have 3.8 - need to update
 # Install Jax and other dependencies on the Compute head node
 head_setup_commands:
-  - conda create -y -n "ray" python=3.8.5 && sudo update-alternatives --install /opt/conda/bin/python python /opt/conda/envs/ray/bin/python 10 && sudo update-alternatives --install /opt/conda/bin/pip pip /opt/conda/envs/ray/bin/pip 10
-  - export PATH="$PATH:/opt/conda/envs/ray/bin" && echo 'export PATH="$PATH:/opt/conda/envs/ray/bin"' >> ~/.bashrc
+  - sudo chown -R $(whoami) /opt/conda/*
+  - conda create -y -n "ray" python=3.8.5
+  - conda activate ray && echo 'conda activate ray' >> ~/.bashrc
+  - python -m pip install --upgrade pip
   - python -m pip install --upgrade "jax[cpu]==0.2.14" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```

Comment:
```
python -m pip install --upgrade "jax[cpu]==0.2.14"
```

It seems like `jax[cpu]` is a typo -- did you mean `jax[tpu]`?

Reply:
`jax[cpu]` is correct. But since it's not a TPU, you may want to remove `-f https://storage.googleapis.com/jax-releases/libtpu_releases.html` since libtpu can't be installed on non-TPU VMs anyway.